### PR TITLE
Call the callback with `this` set to the stream instance

### LINKED ISSUE #3: Call the callback with this as the stream
This would reduce the allocation of one more callback for some standard stuff.

Would you mind if I send a PR on this?

### PR DESCRIPTION
**Pull request by mcollina**
Closes #3.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -10,63 +10,72 @@ var ws = fs.createWriteStream('/dev/null');
 eos(ws, function(err) {
 	expected--;
 	assert(!!err);
+  assert(this === ws);
```
Comment (JakeChampion) [inline test.js]: indentation has gone a bit wrong on this line

Comment (JakeChampion) [inline test.js]: The file looks to use tabs instead of spaces for indentation

**Comment by mcollina** [comment]
@JakeChampion I've just force-pushed a fix, good spot!

**Comment by mafintosh** [comment]
wooooot, 1.2.0

**Comment by mcollina** [comment]
Fantastic!!!

**Pull request merged**